avionics/common/include/rocket.h

```diff
@@ -41,6 +41,24 @@ class CommandReceiver {
   private:
     Rocket &rocket_;
 
+    enum PacketTypeId {
```

Comment:
There's already `command_t` defined in `radio.h`. It would be best if we only had 1 type to represent this, that way also we aren't comparing different types in the switch.

I'd recommend you move this enum into `radio.h` and replace the old definition. If you keep the old name, you wont have to change the occurrences in any other files (or you can use your new name and change it everywhere).

Also you can force the enum to the correct size like this:
```suggestion
    enum PacketTypeId  : uint8_t {
```